resources/views/components/confirmation-modal.blade.php

```diff
@@ -1,6 +1,6 @@
-@props(['id' => null, 'show', 'maxWidth'])
+@props(['id' => null, 'maxWidth' => null])
 
-<x-jet-modal :id="$id" :show="$show" :max-width="$maxWidth ?? '2xl'">
+<x-jet-modal :id="$id" :maxWidth="$maxWidth" {{ $attributes }}>
```

Comment:
The default of `$maxWidth ?? '2xl'` is already declared in the underlying modal component.

I figure it's best to forward all props as well as `$attributes`.